lib/portrayal/schema.rb

```diff
@@ -71,14 +65,14 @@ def freeze
 
       def initialize_dup(source)
         self.class.portrayal.attributes(source).each do |key, value|
-          instance_variable_set('@' + key.to_s, value.dup)
+          instance_variable_set("@\#{key}", value.dup)
```

Comment:
I don't think you want to escape the hash mark you're using for interpolation when setting the ivar name, right? Ditto down below.

Reply:
Ah, the diff makes it hard to recognize but that's [intentional](https://github.com/scottscheapflights/portrayal/blob/65b1e5935c8f40c3d345f6ffc3d83c29a1664e56/lib/portrayal/schema.rb#L49-L80).

Reply:
Oof, my bad. I wasn't accounting for the effect of the heredoc and when the value is interpolated. Never mind me.

Reply:
@phillipgray Although now that you point it out, why is that whole string not a frozen const? 🤔 Gonna change.